setup.cfg

```diff
@@ -13,4 +13,32 @@ flake8-ignore = E133 E203 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
 log_level = DEBUG
 
 [metadata]
+name= ts-cbp
 version = attr: setuptools_scm.get_version
+license=GPL
+project_urls=
+    Bug Tracker=https://jira.lsstcorp.org/secure/Dashboard.jspa
+    Source Code=https://github.com/lsst-ts/ts_CBP
+
+[options]
+package_dir=
+    =python
+packages=find_namespace:
+scripts=
+    bin/run_cbp.py
```

Comment:
Please use script entry points. Python scripts should not be specified as scripts. Entry points are the modern way of doing it. It requires that your asyncio loop goes into a package function and then you do something like:

```
[options.entry_points]
console_scripts =
  butler = lsst.daf.butler.cli.butler:main
```
which is the butler command (also, it's a good convention to not install scripts with a .py suffix -- the user does not care that the command was written in python)